apiary.apib

```diff
@@ -476,6 +476,8 @@ Get the current authenticated user.
                     + tr-TR
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
+            + preferences (object)
+                + invoiceable: true (boolean)
```

Comment:
Q: should we make it clear what is invoiceable?
And maybe add that this is the default value?

Reply:
Not sure if this is necessary. I think it's pretty clear, right?

Reply:
If it's not clear, we should probably give it a better name.